### PR TITLE
Add documentation on regression fields

### PR DESCRIPTION
**Pull request by emceeaich**

### CODE REVIEW

File: _policies/regressions.md [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
-published: false
+published: true
 description: Marking-up regressions
 ---
 
-Page under development.
\ No newline at end of file
+# What is a regression?
+
+A regression is a bug (in our scheme a `defect`) introduced by a change set.
```
Comment (pascalchevrel) [inline _policies/regressions.md]: changeset (see https://en.wikipedia.org/wiki/Changeset)

File: _policies/regressions.md [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+
+- **Bug Type** is `defect`
+- **Keywords** include `regression`
+- **Status_FirefoxNN** is `affected` for each version (in current nightly, beta, and release) of Firefox in which the bug was found
+- The bug's description covers previously working behavior which is no longer working [ed. I need a better phrase for this]
+
+Until the change set which caused the regression has been found through [mozregression](https://mozilla.github.io/mozregression/) or another bisection tool, the bug should also have the `regressionwindow-wanted` keyword. 
+
+Once the change set which caused the regression has been identified, remove the `regressionwindow-wanted` keyword and set the **Regressed By** field to the id of the bug associated with the change set.
+
```
Comment (pascalchevrel) [inline _policies/regressions.md]: We also need to NeedInfo the author of the regressing patch in the bug and ask him to fix the regression.

File: _policies/regressions.md (hunk 1 above)
Comment (emceeaich) [reply _policies/regressions.md]: Adding definition and fixing spelling.

File: _policies/regressions.md (hunk 2 above)
Comment (emceeaich) [reply _policies/regressions.md]: Added

**Pull request merged**